### DISCUSSION TITLE
!KC commands for things that use billboard killcounts. Rune Dragons / Demonic Gorillas for example.

### DISCUSSION THREAD

**Discussion by LastRaidBoss** [runelite/runelite#17773] [Ideas]
Title.

I'll paypal u 5 bucks if u add this...

**Comment by Nightfirecat** [comment]
We typically do not add `!kc` trackers for things which cannot update in real time.

**Reply by LastRaidBoss** [reply]
Yeah but you don't even know what your kc is till you check either, since it doesn't show for chat as you said. So real time updating doesn't matter all that much for those kinda kc logs.

**Reply by Alexsuperfly** [reply]
Except that people are going to kill them and be confused when the chat command doesn't update, this already happens with boss kcs because people do them on mobile, so having the problem exist all the time instead of only desyncing when not using runelite will exacerbate the issue

**Reply by StephanBijzitter** [reply]
Maybe that could be resolved by showing a line in chat below the KC (visible only to the player who typed the command) with instructions on how to update it.

`!kc rune dragons` ->
> Rune Dragons killcount: 1337
> Not accurate? Read the Mural in the Lithkren Vault to update your killcount.

Could be done for bosses that do update in real time as well, but definitely defaulted to once per session:
>Got some kills without RuneLite? Check the Boss Log on a Ring of wealth or in your Player Owned House to update your killcounts.

**Reply by LastRaidBoss** [reply]
I mean for rune dragons you literally have to check the board to know what your kc is anyways is how I see it. demonics you can check collection log/gravestone. I don't think many will find it confusing anymore then needing to boss logs to update old boss killcounts after a long break which breaks it sometimes.

**Reply by cdfisher** [reply]
I think you're significantly underestimating how confusing the need to check the Boss Log is for quite a number of people, so saying it's comparable to that means it would still be something that regularly comes up in #support on Discord.

**Reply by Nightfirecat** [reply]
The major difference between the boss log and demonics/rune dragons is that the former has killcount messages which would automatically update the saved killcounts even without checking the log; _the latter can only be updated through those interfaces_, which is not something we want to support.
